src/main/java/frc/robot/classes/Auton.java

```diff
@@ -120,22 +123,46 @@ private Command resetOdometry(String initialTrajectory) throws NullPointerExcept
         );
     }
 
-    private Command highScoreSequence() {
+    private Command highScoreSequenceCone() {
         return new SequentialCommandGroup(
+        new InstantCommand(intakeSubsystem::setConeMode),
         new ParallelDeadlineGroup(
-            new SequentialCommandGroup(
-            armSubsystem.goToState(ArmState.HIGHTRANSITION),
-            armSubsystem.goToState(ArmState.HIGHSCORE)
-            ), 
+            armSubsystem.goToState(ArmState.HIGHSCORE),
             new StartEndCommand(intakeSubsystem::pull, intakeSubsystem::stop, intakeSubsystem)
         ),
+        new WaitCommand(1).deadlineWith(new RepeatCommand(new InstantCommand(armSubsystem::updateMotors))),
```

Comment:
Why wait for 1 second? Could we reduce it?

Reply:
updatemotors should be out of auton and in the subsystem after we collapse all of the pull requests.  

Should we just ignore this, and fix it when all the pull requests are in?

Reply:
Yeah, fix it later

Reply:
Working on reducing this in current auton branches.